### PR TITLE
KSS plugin

### PR DESCRIPTION
**Pull request by giladgray**
replace postcss deps with kss
remove remarkable typings
write kss plugin

### CODE REVIEW

File: src/plugins/kss.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+/** A single modifier for an example. */
+export interface IKssModifier {
+    documentation: string;
+    name: string;
+}
+
+/**
+ * A markup/modifiers example parsed from a KSS comment block.
+ */
+export interface IKssExample {
```
Comment (themadcreator) [inline src/plugins/kss.ts]: given that the method that returns this is `convertSection` and using the name "example" might be confusing with the `@example` tag, i vote this should be named `IKssSection`

File: test/pluginsTests.ts [hunk 2]
```diff
@@ -52,15 +52,4 @@ describe("Plugins", () => {
         expect(page.contents).with.lengthOf(3);
         expect((page.contents[2] as any).tag).to.equal("othertag");
     });
-
```
Comment (themadcreator) [inline test/pluginsTests.ts]: What, no KSS test?

Comment (themadcreator) [inline test/pluginsTests.ts]: Does KSS require actual files like the typescript plugin? Cuz if so then don't bother

Comment (giladgray) [inline test/pluginsTests.ts]: what do you mean "require actual files"?

Comment (themadcreator) [inline test/pluginsTests.ts]: like tsdoc can't just parse a string, so we can't just use a hardcoded string like we were for the markdown and CSS tests.

File: src/plugins/kss.ts (hunk 1 above)
Comment (giladgray) [inline src/plugins/kss.ts]: that's a private internal method, and it converts a KssSection to a KssExample.

i'm not exposing full "sections" here (which have _tons_ of extra fields), rather a minimal "example" interface tuned for rendering markup/modifiers.

**Comment by themadcreator** [comment]
Huh, circle had `Error uploading artifacts`. I'm gonna kick it again

File: src/plugins/kss.ts (hunk 1 above)
Comment (themadcreator) [inline src/plugins/kss.ts]: Alright

**Pull request merged**